crates/languages/src/python/highlights.scm

```diff
@@ -1,6 +1,59 @@
 (parameter (identifier) @variable)
 (attribute attribute: (identifier) @property)
 (type (identifier) @type)
+(generic_type (identifier) @type)
+
+; Union type X | Y (up to 8 types)
+(type
```

Comment:
@maxbrunsfeld Do you have thoughts on this approach?

Reply:
I don't think we should land this code. I get that it's working around a limitation in the query system right now, but I think we just need to fix that limitation.

I worry this is going to have a negative performance impact, and it's also just hard to understand and maintain.